client/js/components/DailyReportModal/DailyReportModalContainer.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import UserContext from "../../context/UserContext/UserProvider";
+import DailyReportModal from "./DailyReportModal";
+import { graphql, compose } from "react-apollo";
+import { CreateDailyReport } from "../../apollo";
+import { CreateChallengeMutation } from "../../apollo";
+import { UpdateChallengeMutation } from "../../apollo";
+import { AllChallengesQuery } from "../../apollo";
```

Comment:
you can import all of your queries in one line.
`import {CreateDailyReport, UpdateChallengeMutation, etc } from '../../apollo'`